src/core/include/openvino/op/fake_convert.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+
+namespace ov {
+namespace op {
+namespace v13 {
+/// \ingroup ov_ops_cpp_api
+class OPENVINO_API FakeConvert : public Op {
+public:
+    OPENVINO_OP("FakeConvert", "opset13");
+
+    FakeConvert() = default;
+    FakeConvert(const ov::Output<ov::Node>& arg,
+                const ov::Output<ov::Node>& scale,
+                const ov::Output<ov::Node>& shift,
+                std::string destination_type = "f8e4m3",
+                bool apply_scale = false);
```

Comment:
No reason to have `apply_scale` false, suggesting to set it automatically in case if scale input is provided. If one wants to provide shift but doesn't apply scale, then as we don't have "gaps" in the list of operation arguments, one should provide arbitrary scale (will be ignored), then shift and set apply_scale to false. This is only (and looks like very rare situation) when apply_scale should have manual control as a parameter of constructor.

Reply:
FakeConvert(input) ==> apply_scale = false (as not any scale provided)
FakeConvert(input, scale) ==> apply_scale = true
FakeConvert(input, scale, shift) ==> apply_scale = true
FakeConvert(input, scale, shift, "f8e4m", false) ==> apply_scale = false, scale input is ignored

Reply:
That's why I started the conversation above: https://github.com/openvinotoolkit/openvino/pull/20930#discussion_r1385036945
So the final suggestion is to have three constructors and the `apply_scale` attribute.

Reply:
As discussed, in the updated version `scale` input is required and `shift` input optional.
The `apply_scale` attribute has been removed.